src/lib.rs

```diff
@@ -197,3 +199,8 @@ impl Tracer {
         Ok(())
     }
 }
+
+#[test]
+fn test_make_map_filename_0000() {
```

Comment:
What's the `_0000` for?

Reply:
It's a unique ID. Imagine we had lots of tests for that function.

Reply:
Like how we did it in Krun:
https://github.com/softdevteam/krun/blob/30d9c6b4d57b497dc3eb13e02ddc72c911223ce2/krun/tests/test_util.py#L139-L230